Beheerlandschap.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Beheer"
+date: '10-5-2019'
+weight: 100
+---
+
+*Versie 0.1*
+
+## API Beheerlandschap
+
+Het beheren van de API standaarden van VNG Realisatie behelst meer dan alleen het beheer van alle componenten in de GitHub omgeving van VNG Realisatie. Het landschap bevat meerdere omgevingen waar API Beheer een rol te spelen heeft. Naast de GitHub omgeving onderkennen we nog de volgende omgevingen:
+
+- ref.tst.vng.cloud
+- hub.docker.com
+- API testvoorziening
+- test.developer.overheid.nl
+
+De relatie tussen deze omgevingen is gevisualiseerd in de onderstaande image:
+
+![API Beheerlandschap](https://github.com/VNG-Realisatie/api-beheer/blob/melsk-r-beheerlandschap/API%20Beheerlandschap.jpg)
+
+In GitHub bestaat voor elk registratie component een repository maar ook voor enkele andere componenten. Zo hebben we nog een repository voor 'Zaken volgens GEMMA 2.0' dat dient als portaal naar alle gerelateerde GitHub repositories.
+Daarnaast bestaan er ook nog repositories voor de API testvoorziening, voor een tutorial over het werken met GitHub en voor API Beheer (waar dit document deel van uitmaakt).
+
+Het daadwerkelijk vervaardigen van de API standaarden en coderen van de Referentie Implementaties gebeurd in de GitHub repositories van de registratiecomponenten.
+Voor het publiekelijk kenbaar maken van de API standaarden en bijbehorende tooling zijn enkele andere omgevingen in gebruik.
```

Comment:
Enkele andere omgevingen: beheren jullie die niet? of wel?

Reply:
Welke omgevingen bedoel je dan Jan Willem.
Je voorstel heb ik inmiddels op een iets andere wijze gehonoreerd.

Reply:
In de tekst staat "enkele andere omgevingen" . Het is niet duidelijk welk dat zijn en wie die beheert.